components/collections/codepointtrie_builder/Cargo.toml

```diff
@@ -27,14 +27,28 @@ include = [
 [package.metadata.workspaces]
 independent = true
 
+[features]
+# Use the wasm builder
+default = ["wasm"]
+wasm = ["wasmer", "wasmer-wasi"]
+# Use the ICU4C builder
```

Comment:
nit: these comments are not rendered on docs.rs. Put the feature instructions in lib.rs